events/2020/07-contributor-summit/README.md

```diff
@@ -1 +1,51 @@
-# 2020 Kubernetes Contributor Summit China
\ No newline at end of file
+# 2020 Kubernetes Contributor Summit China
```

Comment:
```suggestion
# 2020 Kubernetes Contributor Summit CN
```
super small non-blocking nit 😬: 2 letters are used for NA and EU. I'd either spell out the others or use the 2 letter acronym for all.

Reply:
Good point! I added your suggestion 👍